scripts/tests/scripts_regression_tests.py

```diff
@@ -869,10 +887,46 @@ def test_wait_for_test_cdash_kill(self):
             xml_contents = open(xml_file, "r").read()
             self.assertTrue(r'<TestList><Test>Test_0</Test><Test>Test_1</Test><Test>Test_2</Test><Test>Test_3</Test><Test>Test_4</Test><Test>Test_5</Test><Test>Test_6</Test><Test>Test_7</Test><Test>Test_8</Test><Test>Test_9</Test></TestList>'
                             in xml_contents)
-            self.assertTrue(r'<Test Status="failed"><Name>Test_5</Name>' in xml_contents)
+            self.assertTrue(r'<Test Status="notrun"><Name>Test_5</Name>' in xml_contents)
 
             # TODO: Any further checking of xml output worth doing?
 
+    ###########################################################################
+    def live_test_impl(self, testdir, expected_results, last_phase, last_status):
```

Comment:
Add new "live" tests for wait_for_test testing. These tests handle dynamic TestStatus files instead of static.